experiments/full_fov_comparison.py

```diff
@@ -17,6 +18,14 @@
 from experiment_prototype.experiment_prototype import ExperimentPrototype
 
 
+def widebeam_2antenna(frequency_khz, tx_antennas, antenna_spacing_m):
```

Comment:
Might be a bit misleading with the name, since this depends upon tx_antennas having a list of only two antennas

Reply:
Yeah that's a good call. This more generally could do a 3- or 4-antenna widebeam pattern, or boresight transmission